src/main/java/org/infinispan/persistence/redis/RedisStore.java

```diff
@@ -37,8 +51,13 @@
     @Override
     public void init(InitializationContext ctx)
     {
-        RedisStore.log.info("Redis cache store initialising");
+        RedisStore.log.info("Initialising Redis store for cache " + ctx.getCache().getName());
```

Comment:
Just a note for the future: it would be nice if info and above logging should use jboss logging loggers for i18n.